Rest/Rest/config.xml

```diff
@@ -0,0 +1,12 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<suiteConfig>
+  <ApplicationUnderTest>
+    <AddAssembly>Rest.dll</AddAssembly>
+    <AddNamespace>Rest</AddNamespace>
```

Comment:
Add namespace rest.ContentHandlers

Reply:
ContentHandlers object is also under namespace _**Rest**_.  Verified that no need to add to config.xml file.